pkg/ccl/backupccl/restore_job.go

```diff
@@ -3219,7 +3239,22 @@ func sendAddRemoteSSTs(
 			}
 		}
 	}
-	return nil
+
+	log.Infof(ctx, "creating job to track downloads in %d spans", len(downloadSpans))
+
+	downloadJobRecord := jobs.Record{
+		Description: fmt.Sprintf("Background Data Download for %s", job.Payload().Description),
+		Username:    job.Payload().UsernameProto.Decode(),
+		Details:     jobspb.RestoreDetails{DownloadSpans: downloadSpans},
+		Progress:    jobspb.RestoreProgress{},
+	}
+
+	return execCtx.ExecCfg().InternalDB.DescsTxn(ctx, func(
+		ctx context.Context, txn descs.Txn,
+	) error {
+		_, err := execCtx.ExecCfg().JobRegistry.CreateJobWithTxn(ctx, downloadJobRecord, job.ID()+1, txn)
```

Comment:
Is the +1 merely to clarify that this restore job is related to the first one?

Reply:
yeah, just to make it easy to find. The description says it is related, so we could pick a new random ID, but this seemed nice?